server/storage/endpoint/key_path.go

```diff
@@ -136,3 +140,21 @@ func KeySpaceSafePointPrefix() string {
 func KeySpaceGCSafePointSuffix() string {
 	return "/" + keySpaceGCSafePointSuffix
 }
+
+// KeyspaceMetaPrefix returns the prefix of keyspaces' metadata.
+// Prefix: keyspaces/meta/
+func KeyspaceMetaPrefix() string {
+	return path.Join(keyspacePrefix, keyspaceMeta) + "/"
+}
+
+// KeyspaceMetaPath returns the path to the given keyspace's metadata.
+// Path: /keyspaces/meta/{space_id}
+func KeyspaceMetaPath(spaceID uint32) string {
+	idStr := strconv.FormatUint(uint64(spaceID), spaceIDBase)
+	return path.Join(KeyspaceMetaPrefix(), idStr)
+}
+
+// KeyspaceIDPath returns the path to keyspace id from the given name.
+func KeyspaceIDPath(name string) string {
+	return path.Join(keyspacePrefix, keyspaceID, name)
```

Comment:
Isn't it /keyspaces/id/{name}?

Reply:
yes it is, `keyspaceID` here is a constant.
I'll add some comment here